templates/new/config/config.exs

```diff
@@ -13,8 +13,33 @@ config :nerves, :firmware, rootfs_overlay: "rootfs_overlay"
 # docs for separating out critical OTP applications such as those
 # involved with firmware updates.
 config :shoehorn,
-  init: [:nerves_runtime],
-  app: Mix.Project.config()[:app]
+  init: [:nerves_runtime<%= if init_gadget? do %>, :nerves_init_gadget<% end %>],
+  app: Mix.Project.config()[:app]<%= if init_gadget? do %>
+
+# Uncomment to authorize the device to receive firmware using your public key.
+# See https://hexdocs.pm/nerves_firmware_ssh/readme.html for more information
+# on configuring nerves_firmware_ssh.
+ 
+# config :nerves_firmware_ssh,
```

Comment:
I'd be fine with all of this stuff uncommented. Were you worried about something here?

Reply:
oh, sorry, this is used for both the init_gadget and no init_gadget case.... For the init_gadget case, it would be nice to have all of this committed in by default, but I understand why you did what you did.

Reply:
These extra configs will only render if `--init-gadget` was passed. We can uncomment things, but I am concerned that we are using `File.read!` which will raise an error if the computer does not have any public keys

Reply:
That's a feature. :) If ssh doesn't work, you'll be really sad, so the logic is that getting a compile-time error is good. Perhaps the error should be improved...

Reply:
How do you feel about this in the config

```
key = Path.join(System.user_home!, ".ssh/id_rsa.pub")
unless File.exists?(key), do: 
  Mix.raise("No SSH Keys found. Please generate an ssh key")

config :nerves_firmware_ssh,
  authorized_keys: [
    File.read!(key)
  ]
```

Reply:
:+1: